calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,150 @@
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI.
+
+## Before you begin
+
+**Required**
+
+- Valid registry credentials to set up handshake with the registry scanner
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Docker, using a container runtime
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+- If you use the registry scanner with Docker, only tagged images are scanned. However, if you use the scanner with Amazon AWS or Azure, all images (tagged and untagged) are scanned.
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Set up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4`.
+
+### Set up registry scanner
+
+The registry scanner requires authentication to access a registry. You can use Docker credentials, or Azure/AWS credentials.
+
+#### Docker credentials
+
+Scanning a registry requires valid Docker credentials using these environment variables: `DOCKER_USERNAME` and `DOCKER_PASSWORD`.
+The registry scanner scan images only with tags; untagged images residing in your image registry are not pulled and scanned.
+
+If you have a valid /.docker/config.json, you can also mount this config file on the container while running the registry scanner.
+
+```
+docker run -e ... -v ~/.docker/config.json:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
+```
+#### Azure and AWS credentials
+
+The registry scanner also supports authentication using AZURE and AWS credentials. The registry scanner will retrieve all images without tags.
+
+**Azure credentials required**
+- `AZURE_CLIENT_ID`
+- `AZURE_CLIENT_SECRET` or `AZURE_FEDERATED_TOKEN`
+- `AZURE_TENANT_ID`
+
+**AWS credentials required**
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_REGION`
+
+The registry scanner currently scans one registry at a time. If you specify the Azure or AWS credentials, the Docker username and password will not be used.
+
+#### Required inputs
+
+The registry scanner requires:
+
+- `REGISTRY` - the registry you want to scan. (gcr.io)
+- `IMAGE_ASSURANCE_API_URL` - Get the URL in the Manager UI.
+- `IMAGE_ASSURANCE_API_TOKEN` - Get the token in the Manager UI.
+- Either the `DOCKER_USERNAME` and `DOCKER_PASSWORD`, Azure credentials, or AWS credentials.
```

Comment:
For clarity, it would be nice to see an "or" between "`DOCKER_USERNAME` and `DOCKER_PASSWORD`" and  "Azure credentials".

Reply:
Fixed.